hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/bucket/RemoveAclBucketHandler.java

```diff
@@ -92,8 +92,9 @@ public Void call() throws Exception {
     boolean result = client.getObjectStore().removeAcl(obj,
         OzoneAcl.parseAcl(acl));
 
-    System.out.printf("%s%n", JsonUtils.toJsonStringWithDefaultPrettyPrinter(
-        JsonUtils.toJsonString("Acl removed successfully: " + result)));
+    System.out.printf("%s%n", result ? "ACL removed successfully" :
+        "ACL not removed");
```

Comment:
RemoveAcl returns false, when passedAcl is not existing, then it cannot be removed. Do we need to reword this to convey this?

Reply:
I reverted to previous print messages (fixing add/set/remove part only). 
But I think that statement also does not convey the message properly. If we are trying to add an already existing ACL, shouldn't we return true? And same for remove also. 
We can open a new Jira to work on that if you agree.

Reply:
From my understanding, addAcl behavior is if acl is added successfully returns true, it will return false when acl trying to be added already exists. 

> 
If we are trying to add an already existing ACL, shouldn't we return true?

I think returning true is not right behavior, as it will not be clear whether add is successful or not. We should have returned with clear message to end user, what is differenece between true/false. 

`But I think that statement also does not convey the message properly. `

Agreed this was existing behavior, if you want to fix in a new Jira I am okay with that.